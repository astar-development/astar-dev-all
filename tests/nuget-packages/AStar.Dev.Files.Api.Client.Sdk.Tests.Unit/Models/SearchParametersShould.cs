using AStar.Dev.Files.Api.Client.SDK.Models;

namespace AStar.Dev.Files.Api.Client.Sdk.Models;

public sealed class SearchParametersShould
{
    [Fact]
    public void ReturnTheExpectedToString() =>
        new SearchParameters().ToString().ShouldMatchApproved();

    [Fact]
    public void ReturnTheExpectedToQueryString() =>
        new SearchParameters().ToQueryString().ShouldMatchApproved();
}
