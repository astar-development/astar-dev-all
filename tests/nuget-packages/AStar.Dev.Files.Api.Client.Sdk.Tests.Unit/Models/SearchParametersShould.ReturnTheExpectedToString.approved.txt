{
  "searchFolder": "",
  "searchType": 0,
  "recursive": true,
  "includeSoftDeleted": false,
  "includeMarkedForDeletion": false,
  "currentPage": 1,
  "itemsPerPage": 10,
  "maximumSizeOfThumbnail": 850,
  "maximumSizeOfImage": 1500,
  "sortOrder": 0,
  "searchText": null,
  "excludedViewSettings": {
    "excludeViewedPeriodInDays": 7,
    "excludeViewed": true
  }
}